docs/source/python/development.rst

```diff
@@ -414,3 +416,21 @@ After these steps are completed, the documentation is rendered in HTML
 format in ``docs/_build/html``.  In particular, you can point your browser
 at ``docs/_build/html/index.html`` to read the docs and review any changes
 you made.
+
+
+Building with Docker
+--------------------
+
+You can use Docker to build the documentation:
+
+.. code-block:: shell
+
+  docker-compose build cpp 
+  docker-compose build python 
+  docker-compose run docs 
+
+
+The final output is located under::
```

Comment:
There is also final output located in `docs/_build/html`. The copy at `site/asf-site/docs/latest` is only relevant for uploading to the website. I would prefer that users look in the `docs/` folder.

Reply:
fixed